codec-agnostic-rtp-payload-format.md

```diff
@@ -44,23 +45,24 @@ informative:
 Introduction
 ============
 
-The objective of this spec is to create a generic RTP packetization format that can be used with any video codec or encrypted content and that allows SFUs to perform layer selection without requiring access to the codec payload.
+The objective of this spec is to create a generic RTP packetization format that can be used with any audio or video codec or encrypted content and that allows SFUs to perform routing and layer selection without requiring access to the codec payload.
 
 RTP packetization
 =======================
 
-A generic packetizer, by design, is not expected to understand the format of the media to transmit.
-The unit used by the packetizer to do processing is called a frame in the remainder of the document.
+A generic packetizer, by design, is not expected to understand the format of the media to transmit. The unit used by the packetizer to do processing is called a frame in the remainder of the document.
 
-It is the responsibility of the application using the packetizer to group media content in meaningful frames.
-In the common case of a video codec, the packetizer frame is the frame in byte format (h264 annex b for example) generated by the encoder.
-The packetizer can fragment, in a content-agnostic way, the frame in multiple RTP packets to ensure packets do not exceed the network maximum transmission unit.
+It is the responsibility of the application using the packetizer to group media content in meaningful frames. In the common case of a video codec, the packetizer frame is the frame in byte format (h264 annex b for example) generated by the encoder. 
 
-In the case of a video codec supporting spatial scalability, each spatial layer is split in its own frame.
-The spatial layers are sent in ascending order, with the same RTP timestamp, and only the last RTP packet of the last spatial layer frame will have the marker bit set to 1.
+When the packetizer receives a frame from the application, it MUST fragment the frame content in multiple RTP packets to ensure packets do not exceed the network maximum transmission unit. The content of the frame will be treated as a binary blob by the packetizer, so the decision about the boundaries of each fragment is decided arbitrarily by the packetizer. The packetizer MUST NOT modify the frame content and the concatenating the RTP payload of the RTP packets for each frame MUST produce the exact binary content of the input frame content.
```

Comment:
I am not sure what we cover with 'The packetizer MUST NOT modify the frame content '.
Typo: s/the concatenating/concatenating/

Reply:
I have added also that relying servers must not modify the content as well.